### DISCUSSION TITLE
zotero crashed

### DISCUSSION THREAD

**Discussion by Fututu-h** [windingwind/zotero-better-notes#597] [Q&A]
在zotero中使用Better notes插件时，当我在笔记工作区打开所有笔记可以正常查看、编辑；但阅读文献时在侧栏打开内容较多的笔记就会直接闪退。无论是zoteo6 还是7。
尝试过重新安装软件、插件，重启电脑都不能解决问题。

**Comment by windingwind** [comment]
Please report bugs to [the issue.](https://github.com/windingwind/zotero-better-notes/issues)
